packages/ckeditor5-ui/src/toolbar/block/blocktoolbar.ts

```diff
@@ -148,6 +146,20 @@ export default class BlockToolbar extends Plugin {
 	 */
 	public init(): void {
 		const editor = this.editor;
+		const t = editor.t;
+
+		const editBlockText = t( 'Click to edit block' );
+		const dragToMoveText = t( 'Drag to move' );
+		const editBlockLabel = t( 'Edit block' );
+
+		const isDragDropBlockToolbarPluginLoaded = editor.plugins.has( 'DragDropBlockToolbar' );
+		const label = isDragDropBlockToolbarPluginLoaded ? `${ editBlockText } \n${ dragToMoveText }` : editBlockLabel;
```

Comment:
Why the space before new line? Does it help to adjust the text?

Reply:
No, a leftover - nice catch :+1: will fix it right away.

Reply:
Fixed in 5ec1cb5